docs/src/results_store.md

```diff
@@ -0,0 +1,74 @@
+# Results
+
+Reefmod Engine Outputs and scenario information are recorded from the ReefMod Engine API
+after each scenario(s) run, not before.
+
+## ResultStore Struct
+
+The results stores holds all model outputs from both counterfactuals and interventions.
+Information about scenarios is stored in the scenario field of the result store. The reps
+field the result store is exactly have the number of scenarios contained in the scenario
+dataframe and results YAX dataset. This is because the api forces counterfactuals to be
+evaluated with every intervention run.
```

Comment:
```suggestion
The results stores holds all model outputs from both counterfactual and intervention runs.
Information about scenarios is stored in the scenario field of the result store. The reps
field the result store is exactly have the number of scenarios contained in the scenario
DataFrame and YAX Dataset holding outcomes. This is because the API forces counterfactuals to be
evaluated with every intervention run.
```

See above comment re standardizing "result store".